src/components/toast/toast.scss

```diff
@@ -52,10 +52,10 @@ md-toast {
   }
 
   /* Transition differently when swiping */
-  &.md-swipeleft,
-  &.md-swiperight,
-  &.md-swipeup,
-  &.md-swipedown {
+  &._md-swipeleft,
```

Comment:
@EladBezalel I assume that this would be changed somewhere else (like in the JS), but I don't see any other references to it in the PR. Was it done by someone else?